test/unittests/UtilityTest.php

```diff
@@ -0,0 +1,393 @@
+<?php
+/**
+ * Utility class tests
+ *
+ * PHP Version 7
+ *
+ * @category Tests
+ * @package  Test
+ * @author   Alexandra Livadas <alexandra.livadas@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+require_once __DIR__ . '/../../php/libraries/Utility.class.inc';
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Unit test for Utility class
+ * @category Tests
+ * @package  Main
+ * @author   Alexandra Livadas <alexandra.livadas@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+class FakeUtility extends Utility 
+{
```

Comment:
Missing comment makes phan checking failed.
    /**
     * Returns a list of sites in the database
     *
     * @param bool $study_site If true only return sites that are
     *                         study sites according to the psc
     *                         table
     *
     * @return array an associative array("center ID" => "site name")
     */